core/services/gateway/handlers/vault/handler.go

```diff
@@ -256,12 +274,44 @@ func (h *handler) HandleNodeMessage(ctx context.Context, resp *jsonrpc.Response[
 }
 
 func (h *handler) handleSecretsCreate(ctx context.Context, ar activeRequest) error {
+	l := logger.With(h.lggr, "method", ar.req.Method, "requestId", ar.req.ID)
 	var secretsCreateRequest SecretsCreateRequest
```

Comment:
This isn't a correct request :/ It should be a batch request

Reply:
Can I support batch request in a follow-up PR?
Don't want to make this one any longer.